example.env

```diff
@@ -1,18 +1,18 @@
-GOTRUE_JWT_SECRET="CHANGE-THIS! VERY IMPORTANT!"
+GOTRUE_JWT_SECRET="m1HIQZxYScoiKkTp1DTv45WFiyt7r92p"
 GOTRUE_JWT_EXP=3600
 GOTRUE_JWT_AUD=api.netlify.com
 GOTRUE_DB_DRIVER=mysql
-DATABASE_URL="root@tcp(127.0.0.1:3306)/gotrue_development?parseTime=true"
+DATABASE_URL="root@tcp(127.0.0.1:3306)/gotrue_development?parseTime=true&multiStatements=true"
 GOTRUE_API_HOST=localhost
 PORT=9999
 GOTRUE_SITE_URL=https://my-jam-site.example.com
-GOTRUE_SMTP_HOST=smtp.mandrillapp.com
+GOTRUE_SMTP_HOST=smtp.sendgrid.net
 GOTRUE_SMTP_PORT=587
-GOTRUE_SMTP_USER=test@example.com
-GOTRUE_SMTP_PASS=super-secret-password
-GOTRUE_SMTP_ADMIN_EMAIL=admin@example.com
+GOTRUE_SMTP_USER=apikey
+GOTRUE_SMTP_PASS=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
```

Comment:
probably don't want to have these values in here

Reply:
true but just a test account. thankfully.

Reply:
How do i remove it anyway?

Reply:
just restore the old version of the file. It will be captured in git history, and that is a pain to remove. Like you said, good thing it is a test account